pl_examples/models/hydra_config_model.py

```diff
@@ -0,0 +1,127 @@
+"""
+Example template for defining a Lightning Module with Hydra
+"""
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+from torch.utils.data import DataLoader
+from torchvision.datasets import MNIST
+
+from pytorch_lightning import _logger as log
```

Comment:
is this a standard Python logger?

Hydra supports it directly, you should define your own logger.
see [this](https://hydra.cc/docs/next/tutorials/basic/running_your_app/logging).

Reply:
This is standard library python logger: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/__init__.py.

Reply:
Why are you using the logger from pl instead of defining a logger in the example?

Reply:
I was just following the previous example, but we can use a defined logger in the example.

Reply:
Since this is another divergence from standard PL practices, please add a comment calling it out.
something like:
```
# Hydra configures the Python logging subsystem automatically.
log = logging.getLogger(__name__)
```

Reply:
done